Assets/Scripts/Weapons/ProjectileGun/Bullet.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Bullet : MonoBehaviour
+{
+    private void Start()
+    {
+        Invoke("DestroyBullet", 10.0f);
+    }
+
+    private void OnTriggerEnter2D(Collider2D collide)
+    {
+        if (collide.gameObject.tag != "Bullet")
+        {
+            Destroy(collide.gameObject);
+        }
+    }
+
+    private void DestroyBullet()
+    {
+        Destroy(gameObject);
+    }

```

Comment:
`DestroyBullet()` is never called, could we delete it?

Reply:
It's used in the invoke function

Reply:
I'll add a comment to make that more clear